src/pages/journal/components/test-outcome/test-outcome.ts

```diff
@@ -2,15 +2,16 @@ import { Component, Input } from '@angular/core';
 import { NavController } from 'ionic-angular';
 import { Store } from '@ngrx/store';
 import { StoreModel } from '../../../../shared/models/store.model';
-import { TestOutcomeStartTest } from './test-outcome.actions';
+import { StartTest } from '../../journal.actions';
 
 @Component({
   selector: 'test-outcome',
   templateUrl: 'test-outcome.html',
 })
 export class TestOutcomeComponent {
+
   @Input()
-  slot: any;
+  slotId: string;
```

Comment:
slotId is a number is the schema

Reply:
Fair point.
I'm not a big fan of having `slotId` as a number, but will change it for consistency.